packaging/flatpak/org.mixxx.Mixxx.json

```diff
@@ -0,0 +1,100 @@
+{
+  "app-id": "org.mixxx.Mixxx",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "6.10",
+  "sdk": "org.kde.Sdk",
+  "command": "mixxx",
+  "rename-icon": "mixxx",
+  "finish-args": [
+    "--socket=wayland",
+    "--socket=fallback-x11",
+    "--socket=pulseaudio",
+    "--device=all",
+    "--share=network",
+    "--share=ipc",
+    "--filesystem=xdg-run/pipewire-0",
+    "--filesystem=xdg-music",
+    "--filesystem=/media",
+    "--filesystem=/run/media",
+    "--system-talk-name=org.freedesktop.UPower",
+    "--system-talk-name=org.freedesktop.RealtimeKit1",
+    "--talk-name=org.freedesktop.ScreenSaver",
+    "--env=ALSA_CONFIG_PATH=",
+    "--metadata=X-DConf=migrate-path=/org/mixxx/Mixxx",
+    "--persist=.mixxx"
+  ],
+  "add-extensions": {
+    "org.freedesktop.LinuxAudio.Plugins": {
+      "directory": "extensions/Plugins",
+      "version": "25.08",
+      "add-ld-path": "lib",
+      "merge-dirs": "lv2",
+      "subdirectories": true,
+      "no-autodownload": true
+    }
+  },
+  "cleanup": [
+    "*.a",
+    "*.la"
+  ],
+  "modules": [
+    "modules/libmad.json",
+    "modules/lv2.json",
+    "modules/serd.json",
+    "modules/zix.json",
+    "modules/sord.json",
+    "modules/sratom.json",
+    "modules/lilv.json",
+    "modules/googletest.json",
+    "modules/benchmark.json",
```

Comment:
We could probably remove those and add `BUILD_BENCH=OFF` and `BUILD_TESTING=OFF`. Since we don't install the `mixxx-test` and `mixxx-benchmark` target, we could remove it for now.

Going forward, we might want to add a variant (Or maybe a YAML patch using [yq Multiply](https://mikefarah.gitbook.io/yq/operators/multiply-merge)?) to allow to build a flatpak image with these targets, so we can test them in CI, but this should be follow up.